lib/internal/streams/utils.js

```diff
@@ -83,7 +83,7 @@ function isWritableEnded(stream) {
 
 // Have emitted 'finish'.
 function isWritableFinished(stream, strict) {
-  if (!isWritableNodeStream(stream)) return null;
```

Comment:
Check for web stream

Reply:
I think i should remove the changes on isReadableFinished and isWritableFinished since those functions wouldnt be reached as per the present logic